core/modules/user/user.admin.inc

```diff
@@ -168,6 +168,57 @@ function user_admin_settings($form, &$form_state) {
     }
   }
 
+  // Password settings.
+  $form['password_policy'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Password policy'),
+    '#weight' => -1,
+  );
+  $output = '';
+  $criteria = array();
+  $criteria[] = t('not the same as the username');
+  $criteria[] = t('not the same as the e-mail address');
+  $criteria[] = t('sufficiently complex: containing a number of lower- and uppercase letters, digits and symbols');
+  $criteria[] = t('at least as many characters as the minimum number set below');
+
```

Comment:
We should remove this now. We have the "inline combo" with the number field visible all the time, so I think that this is superfluous here.